Makefile

```diff
@@ -551,16 +549,11 @@ test_e2e_xdc: bins
 cover_profile:
 	$Q mkdir -p $(BUILD)
 	$Q mkdir -p $(COVER_ROOT)
-	$Q echo "mode: atomic" > $(UNIT_COVER_FILE)
 
 	$Q echo Running special test cases without race detector:
 	$Q go test -v ./cmd/server/cadence/
 	$Q echo Running package tests:
-	$Q for dir in $(PKG_TEST_DIRS); do \
-		mkdir -p $(BUILD)/"$$dir"; \
-		go test "$$dir" $(TEST_ARG) -coverprofile=$(BUILD)/"$$dir"/coverage.out || exit 1; \
-		cat $(BUILD)/"$$dir"/coverage.out | grep -v "^mode: \w\+" >> $(UNIT_COVER_FILE); \
-	done;
+	$Q go test -short ./... $(TEST_ARG) -coverprofile=$(UNIT_COVER_FILE);
```

Comment:
`PKG_TEST_DIRS` is not used anymore that means all go tests including integration tests will be executed, right? We should keep `cover_profile` lightweight and not depend on C* or other db containers being available.
Looks like unit test in CI is failing. Once you make it pass please share coveralls link so we can compare before and after

Reply:
yeah, that's what was preventing me from doing this earlier.

_build_ all packages for `go test ./...`: yes absolutely we should.  integration probably need `t.Skip("explanation")` stuff tho, like we have for cassandra now.  It's just not universally applied iirc.